lib/posix/posix_linux_amd64_consts.nim

```diff
@@ -365,7 +365,7 @@ const SCHED_RR* = cint(2)
 const SCHED_OTHER* = cint(0)
 
 # <semaphore.h>
-const SEM_FAILED* = cast[pointer]((nil))
+const SEM_FAILED* = nil
```

Comment:
`const SEM_FAILED* = pointer(nil)` ?

Reply:
`const a = cast[pointer](nil)` and `const a = nil` are the same I guess.